### PR TITLE
(user) add signup / login / logout and basic front end with angular

### PR DESCRIPTION
**Pull request by raphaeldieu**
Add all user functionnalities login logout signup

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -11,6 +11,11 @@
 // about supported directives.
 //= require angular/angular
 //= require angular-route/angular-route
+<<<<<<< HEAD
```
Comment (f3r) [inline app/assets/javascripts/application.js]: ouch

Comment (h8rry) [inline app/assets/javascripts/application.js]: Ouchie.

File: bower.json [hunk 2]
```diff
@@ -8,5 +8,7 @@
     "angular-route": "~1.4.5",
     "bootstrap": "~3.3.5",
     "flat-ui": "~2.2.2"
```
Comment (f3r) [inline bower.json]: where is my comma?

**Pull request merged**